packages/TCSLib/tcslib.0.5/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A multi-purpose library for OCaml"
+description: "A multi-purpose library for OCaml."
+maintainer: ["Oliver Friedmann" "Martin Lange"]
+authors: ["Oliver Friedmann" "Martin Lange"]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/tcslib"
+bug-reports: "https://github.com/tcsprojects/tcslib/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.8"}
```

The CI doesn't run anything before 4.8 so we generally advise to set this as a lower bound. Do you have specific use-case for going lower? Some tests you are running to check that it works lower than that?